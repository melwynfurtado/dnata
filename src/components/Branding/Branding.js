import React from 'react'

const Branding = () => (
  <nav className="navbar navbar-dark bg-dark">
    <h1><span className="navbar-brand mb-0 h1">MTravels</span></h1>
  </nav>  
)

export default Branding