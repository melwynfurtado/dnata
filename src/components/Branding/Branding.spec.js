import React from 'react'
import renderer from 'react-test-renderer'
import Branding from './Branding'

describe('Branding Component', () => {
  let Component

  beforeEach(() => {
    Component = <Branding />
  })

  it('renders correctly', () => {
    const tree = renderer
                  .create(Component)
                  .toJSON()
    expect(tree).toMatchSnapshot()
  })
})