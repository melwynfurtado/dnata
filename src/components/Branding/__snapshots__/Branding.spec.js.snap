// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branding Component renders correctly 1`] = `
<nav
  className="navbar navbar-dark bg-dark"
>
  <h1>
    <span
      className="navbar-brand mb-0 h1"
    >
      MTravels
    </span>
  </h1>
</nav>
`;
